packages/rx_bloc_test/pubspec.yaml

```diff
@@ -1,25 +1,25 @@
 name: rx_bloc_test
 description: A Flutter package with the goal to enable testing rxBlocs from the FlutterRxBloc package with ease.
-version: 2.0.1
+version: 3.0.0
 repository: https://github.com/Prime-Holding/rx_bloc/tree/master/packages/rx_bloc_test
 issue_tracker: https://github.com/Prime-Holding/rx_bloc/issues
 homepage: https://primeholding.com
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.13.0 <3.0.0"
```

Comment:
@DDavidPrime, do we need to constraint the devs to use 2.13.0, or rx_bloc will be running fine with 2.12 as well?